### ISSUE TITLE
Connect ProtectedSeas catalogue as ODIS node

### ISSUE THREAD

**Issue by jmckenna** [iodepo/odis-arch#319]
- related to https://github.com/iodepo/odis-arch/issues/101

cc @vzett

**Comment by pbuttigieg** [comment]
Pinging this thread- MPA data is of high importance

**Comment by vzett** [comment]
Yes I talked to Lucy yesterday and we should meet again and work out any
final pattern changes so we can implement.

Virgil

**Comment by jmckenna** [comment]
@vzett great to hear

**Comment by jmckenna** [comment]
update: the `protectedAreaData.json` template produced earlier together, now lives in the `odis-in` repo [here](https://github.com/iodepo/odis-in/blob/master/dataGraphs/thematics/dataset/graphs/protectedAreaData.json)
